### ISSUE TITLE
[Bug]: Preview output of HTTP Post does not show data unless you hit Right Arrow

### ISSUE THREAD

**Issue by maxbobse** [apache/hop#3628]
### Apache Hop version?

2.7.0

### Java version?

openjdk version "11.0.15" 2022-04-19

### Operating system

macOS

### What happened?

I am performing a HTTP Post step and the response body is a JSON. When I click on the HTTP Post and select "View output" it shows an empty field for the response column. When I click in the first row cell of that column and hit the Right Arrow Key, it actually shows the JSON data.

This is very misleading because I would expect to see the data right away without having to click in there.

See https://youtu.be/ysgf0dv-3NQ for a demo.

### Issue Priority

Priority: 2

### Issue Component

Component: Hop Gui

**Comment by hansva** [comment]
there probably is a newline in your data and by pressing right button you go to the next line. The preview isn't that great with multiline data so I think that could be improved.

**Comment by maxbobse** [comment]
You are right, there is a line break at the beginning, sorry I missed that. In that case I would not consider this a bug anymore but rather room for improvement in the GUI?

**Comment by hansva** [comment]
Yes, I think we should have a small arrow on multi-line fields to expand them and show the content.
